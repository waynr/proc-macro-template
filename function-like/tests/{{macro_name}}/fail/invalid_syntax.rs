use proc_macro_template::layer;

fn main() {
    {{macro_name}}!(meow);
}
