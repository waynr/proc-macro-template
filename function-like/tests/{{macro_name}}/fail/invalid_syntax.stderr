error: unexpected token
 --> tests/fail/invalid_syntax.rs:4:12
  |
4 |     {{macro_name}}!(meow);
  |                     ^^^^
